Source/WebKit/UIProcess/Cocoa/SOAuthorization/RedirectSOAuthorizationSession.h

```diff
@@ -37,6 +37,9 @@ class RedirectSOAuthorizationSession final : public NavigationSOAuthorizationSes
 
     static Ref<SOAuthorizationSession> create(RetainPtr<WKSOAuthorizationDelegate>, Ref<API::NavigationAction>&&, WebPageProxy&, Callback&& completionHandler);
 
+    void ref() const final { SOAuthorizationSession::ref(); }
```

Comment:
Why isn't this on `NavigationSOAuthorizationSession` ?

Reply:
Fixed!

Reply:
Fixed!

Reply:
Fixed!